bin/docserv-stitch

```diff
@@ -10,8 +10,9 @@
 # INPUT_DIR                       # Directory with config files
 # OUTPUT_FILE                     # Output file name (parent directory must exist)
 #
-# XML tool deps: xmlstarlet, jing, xmllint
-
+# XML tool deps: xmlstarlet, jing, xmllint, xsltproc
+#
+# TODO: Move some checks for xmlstarlet into XSLT stylesheet
```

Comment:
Maybe report as an issue rather than adding a TODO here..?

Reply:
Absolutely right. I've opened issue #110 for this purpose.